### PR TITLE
[doc] expanding the State's docstrings, small cleaning

### PR DESCRIPTION
**Pull request by djarecka**
## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist

- [ ] All tests passing
- [ ] I have added tests to cover my changes
- [ ] I have updated documentation (if necessary)
- [ ] My code follows the code style of this project
(we are using `black`: you can `pip install pre-commit`,
run `pre-commit install` in the `pydra` directory
and `black` will be run automatically with each commit)

## Acknowledgment
- [x] I acknowledge that this contribution will be available under the Apache 2 license.

### CODE REVIEW

**Comment by djarecka** [comment]
@effigies @satra - do you have an experience with this issue? any suggestions? https://app.circleci.com/pipelines/github/nipype/pydra/459/workflows/09db24f5-14bc-41de-958a-eb83f13ae780/jobs/521

I've read that this is a common warning, but not sure why this is here interpreted as an error.

File: pydra/engine/state.py [hunk 1]
```diff
@@ -193,7 +205,9 @@ def left_splitter_rpn(self):
 
     @property
     def left_splitter_rpn_compact(self):
-        # left rpn part, but keeping the names of the nodes, e.g. [_NA, _NB, *]
+        """ the Left Part of the splitter using RPN in a compact form,
+        (without unwrapping the states from previous nodes), e.g. [_NA, _NB, *]
```
Comment (satra) [inline pydra/engine/state.py:209]:
```suggestion
        (without unwrapping the states from previous nodes), e.g. [\_NA, \_NB, \*]
```

these are reserved characters in both restructured text and markdown.

Comment (satra) [inline pydra/engine/state.py:209]:
```suggestion
        r""" the Left Part of the splitter using RPN in a compact form,
        (without unwrapping the states from previous nodes), e.g. [\_NA, \_NB, \*]
```

Comment (djarecka) [reply pydra/engine/state.py:209]: oh, ok, you're right. Thanks!

**Pull request merged**